drizzlepac/haputils/catalog_utils.py

```diff
@@ -268,10 +311,12 @@ def compute_background(self, box_size, win_size,
                                            bkg_estimator=bkg_estimator(),
                                            bkgrms_estimator=rms_estimator(),
                                            exclude_percentile=percentile, edge_method="pad",
-                                           mask=mask)
+                                           mask=self.inv_footprint_mask)
 
                         # Apply the coverage mask to the returned background image
-                        bkg.background *= ~mask
+                        fits.PrimaryHDU(data=bkg.background.astype(np.uint16)).writeto(self.imgname.replace(".fits", '_B_bkg2d.fits'))
```

Comment:
Once again, these should either be diagnostic mode only or removed before merging.  Same with Line 319 as well.